internal/common-operation.go

```diff
@@ -238,6 +240,11 @@ func CreateClientConfig(context *ClientContext) (*sarama.Config, error) {
 		config.Net.SASL.TokenProvider = tokenProvider
 	}
 
+	if _proxy := os.Getenv("ALL_PROXY"); _proxy != "" {
```

Comment:
since `proxy.FromEnvironment()` handles both `ALL_PROXY` and `all_proxy` as names for the env variable, we should probably do the same here